### PR TITLE
Synching matrix

### PR DESCRIPTION
**Pull request by rachaelbradley**

### CODE REVIEW

File: site/src/pages/index.astro [hunk 1]
```diff
@@ -496,6 +508,10 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
           which is insufficient to disambiguate for colorblind users, as some
           controls feature the same icons distinguished only by color.
         </dd>
+        <dt>1.4.1 Use of Color</dt>
+        <dd>The controls are color coded and instructions depend on them. </dd>
+        <dt>1.4.10: Reflow</dt>
+        <dd>The controls disappear when magnified.</dd>
```
Comment (kfranqueiro) [inline site/src/pages/index.astro:514]: This is technically due to the mobile breakpoint being hit, which explicitly hides the controls (it's not that it's being scrolled off-screen). Is reflow still accurate in this case?

**Comment by kfranqueiro** [comment]
Would now also be a good time to re-evaluate 1.1.1 Non-text Content on `museum/collections/`? I flagged last week that I don't think the collections list page actually fails this, because none of the images in the main region even have alt text.

File: site/src/pages/index.astro [hunk 2]
```diff
@@ -397,6 +405,8 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
         <dd>
           Forms in the checkout process time out after an undisclosed period of time.
         </dd>
+        <dt>2.4.3 Focus Appearance</dt>
```
Comment (kfranqueiro) [inline site/src/pages/index.astro:408]: Doesn't this fail Focus Visible rather than Focus Appearance?

(Also, Focus Appearance is 2.4.13, not 2.4.3)

**Comment by rachaelbradley** [comment]
* Would now also be a good time to re-evaluate 1.1.1 Non-text Content on `museum/collections/`? I flagged last week that I don't think the collections list page actually fails this, because none of the images in the main region even have alt text. (Update: I've removed this after doublechecking all images on the page.)

Let's discuss at today's meeting.

File: site/src/pages/index.astro [hunk 3]
```diff
@@ -381,6 +381,34 @@ import MetaLayout from "@/layouts/MetaLayout.astro";
           The first post under "In the News" leads to a 404 page with no clear description of what went wrong.
         </dd>
       </dl>
+
+      <dl slot="wcag2">
```
Comment (kfranqueiro) [inline site/src/pages/index.astro:385]:
```suggestion
      <dl slot="wcag3">
```

**Review by kfranqueiro — APPROVED** [review]
I am good with this if you are, @rachaelbradley (please have a look over the updates I've made too)

**Pull request merged**